### PR TITLE
bold gentle introduction

### PR DESCRIPTION
**Pull request by leeederek**
- [x] author gentle introduction
- [x] add new side bar components
- [x] insert graphics 

Supersedes: https://github.com/OffchainLabs/arbitrum-docs/pull/946

### CODE REVIEW

**Comment by leeederek** [comment]
Hey @symbolpunk - I added 2 of the 3 graphics into the docs and got sign-off from @rauljordan on the content. I *think* this is ready for your review unless you believe you want to wait for the very last graphic.

The very last graphic is just explaining how challenges behave before and after bold. Let me know!

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (symbolpunk) [inline arbitrum-docs/bold/bold-gentle-introduction.md:67]:
```suggestion
With BOLD at its core, Arbitrum charts a course towards being recognized as a Stage 2 rollup by addressing the currently yellow (above) State Validation wedge in [L2Beat's risk analysis pie chart](https://l2beat.com/scaling/summary). BOLD contributes to a more decentralized, efficient, and robust rollup ecosystem. Additionally, L2 Orbit chains launched with BOLD will have permissionless validation available for them out of the box by upgrading their Nitro node software.
```

> L2 Orbit chains launched with BOLD will have permissionless validation available for them out of the box by upgrading their Nitro node software.

It may be worth addressing two audiences specifically here, assuming my understanding is correct: Current Orbit chain operators (who can enable BOLD by upgrading Nitro), and future Orbit chain operators (who will have BOLD enabled by default by way of Nitro). Right now we're combining these two.

Not sure if we want to address whether this BOLD-enabled-by-default Nitro upgrade will be required, or the impact it will have on chain operators (could it be argued that this is a breaking change, or is it a completely unobtrusive 'pure win' from RaaS providers' perspective).

**Review by symbolpunk — APPROVED** [review]
No blocking feedback, just some nits (feel free to commit whatever you agree with; disregard the rest) 👍

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (leeederek) [reply arbitrum-docs/bold/bold-gentle-introduction.md:48]: Yup, acknowledging

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (leeederek) [reply arbitrum-docs/bold/bold-gentle-introduction.md:67]: Thanks - great call out. Will address in a separate commit!

**Comment by leeederek** [comment]
DO NOT MERGE

File: arbitrum-docs/bold/partials/_bold-public-preview-banner-partial.md
Comment (TucksonDev) [inline arbitrum-docs/bold/partials/_bold-public-preview-banner-partial.md:3]:
```suggestion
The BOLD dispute protocol is currently deployed on a public testnet and is tagged as an `alpha` release. The code has not been audited, and should **not be used in production scenarios**. This documentation is currently in [public preview](../concepts/public-preview-expectations.md).
```

Following the style conventions [here, point 10](https://docs.arbitrum.io/for-devs/contribute#style-conventions), we should limit the use of characters like '&' and '/'.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:23]: I think we should mention that the upper time-bound enforced for resolving disputes is added to the current challenge period (e.g., if 3 days have passed since the assertion was posted, and someone initiates a challenge, the full confirmation period will be 10 days)

Reading the text I assumed that I can withdraw my funds after 1 challenge period regardless of whether or not disputes arise, and that's not the case, and that might be a bit confusing.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:26]:
```suggestion
- BOLD is currently considered to be in `alpha` release and is deployed on a public testnet. [Follow this guide](https://github.com/OffchainLabs/bold-validator-starter-kit) to deploy a BOLD validator to test and explore, first hand, how BOLD works to secure Arbitrum chains. To learn more about BOLD, please check out the [BOLD whitepaper](https://github.com/OffchainLabs/bold/blob/main/docs/research-specs/BOLDChallengeProtocol.pdf) and [BOLD's code and specifications on Github](https://github.com/OffchainLabs/bold).
```

[Style convention 10](https://docs.arbitrum.io/for-devs/contribute#style-conventions)

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:24]: Should we also link to the description of the rollup protocol for some extra context?
https://arbitrum-docs-git-derek-bold-gentle-intro-docs-offchain-labs.vercel.app/inside-arbitrum-nitro/#arbitrum-rollup-protocol

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:29]: I don't think the states are confirmed to the chain's DA layer (or at least, I believe DAs are not used that way right now, if I'm not mistaken).

I'd say they are confirmed to the chain's base layer, which acts as the security layer, but maybe someone else from the Product team has a better wording for it.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:31]: Should we use another wording for this?
"secures the process of withdrawing assets" might imply that it was not secure before BOLD, but that's not the case. We are just decentralizing the ability to participate in the validation of the state of the chain.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:39]: Here too, the maximum challenge period is 2*6.4

Comment (TucksonDev) [reply arbitrum-docs/bold/bold-gentle-introduction.md:39]: Also, we might want to change the "L2 to L1" wording to something like "withdrawals to the parent chain", since this will also be applied to Orbit chains.

Is a bit tricky, since we clearly want to emphasize the fact that Arbitrum chains are one of the main beneficiaries of this update, but we probably want to make sure Orbit chains are also benefited from this.

Another thing we could do is to add a note at the end or the beginning of the page stating that although "L2 to L1" language is used, this also applies to any chain settling to a parent chain.

Comment (TucksonDev) [reply arbitrum-docs/bold/bold-gentle-introduction.md:39]: Finally, "a new dispute system that lets a single entity secure Arbitrum", this is the case with the current challenge protocol too, is not something new with Bold. Is one of the main selling points of the current challenge protocol, so I think we shouldn't mention it as a feature of Bold.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:44]: Here too, it would be time-bounded to 2 challenge periods.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:46]: "Instead, because L2 states are completely deterministic and can eventually be proven on Ethereum, any party can stake on this correct, deterministic state and, through interactive fraud proofs, can prove that their claim is correct. This means that a single honest party staking on the correct state assertion will always win, guaranteed."

This is also the case with the current challenge protocol, not a feature from Bold.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [reply arbitrum-docs/bold/bold-gentle-introduction.md:26]: '&' is also used in other parts of the document, can you update all occurrences? 🙏

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:54]: "posting both transaction data & the resulting state from those transactions to a data availability layer"

Same reasoning as before. I'm not 100% sure now, but I would say that the DA only stores transaction data.

Can you verify?

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:60]: "(this window can be decreased by the DAO)"

Is this something we want to mention here?

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:73]: This is all the case with the current challenge protocol, not new with Bold.

Using this text, I'd say that "The BOLD protocol provides the guardrails and rules for how validators challenge claims about the state of an Arbitrum chain" . But the rest of the paragraph is the behavior of the current challenge protocol.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:77]: It doesn't work exactly like this. There's general some misconception around what are the differences between L2 blocks and Rollup blocks (or RBlocks). There's always been that misconception though 😄 

In the challenge protocol, RBlocks are used, which are unrelated to L2 blocks. There's more info in [this FAQ](https://docs.arbitrum.io/learn-more/faq#what-is-the-difference-between-an-l2-block-and-a-rblock) and [this page](https://docs.arbitrum.io/inside-arbitrum-nitro/#arbitrum-rollup-protocol)

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:78]: I would confirm the definition of "edge" given here. This is Bold specific and I'm not completely sure of the right definition.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:83]:
```suggestion
4. **One-step proof:** Once a challenge is isolated to a dispute about a single step, the honest party's edge can be proven correct with a *one-step proof*. A one-step proof is a proof that a single step of computation results in a particular state. The honest party will submit such a proof, whose correctness is easily and quickly validated via execution on Ethereum. The honest edge will thus be confirmed, and the dishonest one rejected.
```

And also, the disputed single step is actually executed on Ethereum, rather than submitted by the honest party. I.e., both parties have submitted what they think is the resulting state after executing that step, and that step is executed by the OneStepProof smart contract on Ethereum (or the parent chain), to be completely unbiased/objective.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (TucksonDev) [inline arbitrum-docs/bold/bold-gentle-introduction.md:127]: This is more a marketing issue, but as a user I would feel more secure without the first "Yes", and reading only the rest of the text.

File: arbitrum-docs/bold/bold-gentle-introduction.md
Comment (leeederek) [reply arbitrum-docs/bold/bold-gentle-introduction.md]: makes sense, how does this sound?

> Once a challenge is isolated down to a dispute about a single step, both parties run that step to produce, and then submit, a *one-step proof* to the `OneStepProof` smart contract on the parent chain (e.g. Ethereum). A one-step proof is a proof that a single step of computation results in a particular state. The smart contract on the parent chain will execute the disputed step to validate the correctness of a submitted proof from the two parties. It is at this point that the honest party's proof will be deemed valid and it's tree of edges will get confirmed, while the dishonest party has their edges rejected.

Comment (TucksonDev) [reply arbitrum-docs/bold/bold-gentle-introduction.md]: Sounds much better, but I think we should have someone else's take on it too, since I'm not sure about the "proof submission" part. I have limited knowledge about this, but in my mind, only initialState and instructionToExecute are sent to the smart contract, and the sc returns the actual state that results from that operation, which will make the honest party win the challenge.

**Comment by leeederek** [comment]
TODO

- [ ] Link the new research paper and AIP once they are published

**Pull request merged**
